docs/content/en/usage.md

```diff
@@ -3,35 +3,91 @@ title: Usage
 description: 'Access text, images, and other media with Nuxt and the Sanity headless CMS.'
 category: Getting started
 position: 4
-version: 0.39
+version: 0.40
```

Comment:
I wouldn't increment the version as this isn't a new feature - just fuller documentation 
```suggestion
version: 0.39
```

Reply:
ah yes, makes sense.